src/PythonConverter/PythonConverterService.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactNode, useEffect, useState } from 'react';
+import { createGenericContext } from '../util/GenericContext';
+import makeAsyncScriptLoader from 'react-async-script';
+import { InputFiles } from '../services/ShSimulatorService';
+
+const PythonConverterUrl = 'https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js';
```

Comment:
add comment
// Method suggested in https://pyodide.org/en/stable/usage/downloading-and-deploying.html

Reply:
```suggestion
// as for now there is no reasonable npm package for pyodide
// method suggested is suggested in https://pyodide.org/en/stable/usage/downloading-and-deploying.html
const PythonConverterUrl = 'https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js';
```